android/player/BUILD

```diff
@@ -1,33 +1,43 @@
-load(":defs.bzl", "main_deps", "main_exports", "main_resources", "test_deps")
+load(":deps.bzl", "main_deps", "main_exports", "main_resources", "test_deps")
 load("@build_constants//:constants.bzl", "VERSION")
+load("@rules_kotlin//kotlin:android.bzl", "kt_android_library")
 load("@rules_kotlin//kotlin:core.bzl", "kt_kotlinc_options")
+load("@rules_kotlin//kotlin:jvm.bzl", "kt_jvm_library")
 load("//jvm:defs.bzl", "distribution")
-load("@grab_bazel_common//tools/databinding:databinding.bzl", "kt_db_android_library")
 load("@rules_player//kotlin:defs.bzl", "kt_jvm_junit5_test", "lint")
 
-kt_db_android_library(
-    name = "player",
+kt_android_library(
+    name = "player_android",
     srcs = glob(["src/main/java/**/*.kt"]),
     custom_package = "com.intuit.playerui.android",
     manifest = ":src/main/AndroidManifest.xml",
     resource_files = glob(["src/main/res/**"]),
     resources = main_resources,
-    tags = ["maven_coordinates=com.intuit.playerui:android:aar:%s" % VERSION],
     visibility = ["//visibility:public"],
```

Comment:
One last thing - since this is now an intermediate target, can we remove the public visibility here to ensure usage of this flows through the end target?